### PR TITLE
142 add easier signature metrics

### PR DESCRIPTION
**Pull request by mgdenno**
* Adds a few new metrics to the queries:
  * annual_peak_relative_bias
  * spearman_correlation
  * kling_gupta_efficiency_mod1
  * kling_gupta_efficiency_mod2

### CODE REVIEW

File: tests/test_duckdb_pandas_tds_compare.py [hunk 1]
```diff
@@ -19,16 +19,18 @@
...
-TDS.insert_joined_timeseries(
+tds.insert_joined_timeseries(
     primary_filepath=PRIMARY_FILEPATH,
     secondary_filepath=SECONDARY_FILEPATH,
     crosswalk_filepath=CROSSWALK_FILEPATH,
     drop_added_fields=True,
 )
 
+# Join the attributes
+tds.join_attributes("/home/matt/repos/teehr/tests/data/test_study/geo/test_attr.parquet")
```
Comment (samlamont) [inline tests/test_duckdb_pandas_tds_compare.py:33]: Looks like the path is hardcoded to your local directory

File: src/teehr/queries/duckdb.py [hunk 2]
```diff
@@ -188,6 +188,8 @@ def get_metrics(
             {tqu._remove_duplicates_mq_cte(mq)}
         )
         {tqu._nse_cte(mq)}
+        {tqu._annual_metrics_cte(mq)}
+        {tqu._spearman_ranks_cte(mq)}
```
Comment (samlamont) [inline src/teehr/queries/duckdb.py:192]: Just for my own knowledge, these are the "pre-calculated" metrics which are used `metrics` CTE below, is that correct?

**Review by samlamont — CHANGES_REQUESTED** [review]
Just noticed a hard coded path to your local directory in `test_duckdb_pandas_tds_compare.py`

File: tests/test_duckdb_pandas_tds_compare.py (hunk 1 above)
Comment (mgdenno) [reply tests/test_duckdb_pandas_tds_compare.py:33]: Good catch.  Will address.

**Review by samlamont — APPROVED** [review]

**Pull request merged**